core/crypto/randomness/impl/randomness_provider.cpp

```diff
@@ -35,4 +35,10 @@ namespace fc::crypto::randomness {
     return Randomness(hash);
   }
 
+  int RandomnessProviderImpl::randomInt(const Randomness &randomness,
+                                        size_t nonce,
+                                        size_t limit) {
+    throw std::string("Not implemented");
```

Comment:
I think it's better to place `TODO ` here instead of `hpp` and please create a task in JIRA for the `TODO`

Reply:
Fixed